src/config/modules.rs

```diff
@@ -0,0 +1,21 @@
+use axum::extract::{Path, State};
+use axum::Json;
+use fedimint_core::api::InviteCode;
+use serde_json::json;
+
+use crate::config::meta::MetaOverrideCache;
+use crate::config::FederationConfigCache;
+
+pub async fn fetch_federation_modules(
+    Path(invite): Path<InviteCode>,
+    State((config_cache, _)): State<(FederationConfigCache, MetaOverrideCache)>,
+) -> crate::error::Result<Json<serde_json::Value>> {
+    let config = config_cache.fetch_config_cached(&invite).await?;
+    let module_names = config
+        .modules
+        .into_iter()
+        .map(|(module_instance_id, module_config)| module_config.kind().as_str().to_owned())
+        .collect::<Vec<_>>();
```

Comment:
If we only care about module kinds we can collect as a `HashMap` here, or do we want to know if there are multiple instances of the same module?